cmd/gke-gcloud-auth-plugin/main.go

```diff
@@ -38,6 +38,10 @@ type gcloudConfiguration struct {
 	X          map[string]interface{} `json:"-"` // Rest of the fields should go here.
 }
 
+var (
+	useAdcPtr = pflag.Bool("use_application_default_credentials", false, "returns exec credential filled with application default credentials.")
```

Comment:
Why pflag instead of native flag?

Reply:
do we prefer `use-application-default-credentials`? I know that gcloud flags use `-` instead of `_`

Reply:
1. I am already using pflag to support k8s build in verflag -> line 47 verflag.PrintAndExitIfRequested(). So, I continued used pflag.
2. I am reusing use_application_default_credentials from gcloud - which is the following..
gcloud config set container/use_application_default_credentials true